.travis.yml

```diff
@@ -26,24 +26,28 @@ env:
 matrix:
   include:
    - php: 5.2
-     env: WP_VERSION=latest
+     env: WP_VERSION=latest WC_VERSION=2.6.14
    - php: 5.2
-     env: WP_VERSION=4.4
+     env: WP_VERSION=4.4 WC_VERSION=2.6.14
    - php: 5.3
-     env: WP_VERSION=latest
+     env: WP_VERSION=latest WC_VERSION=2.6.14
    - php: 5.3
-     env: WP_VERSION=4.4
+     env: WP_VERSION=4.4 WC_VERSION=2.6.14
    - php: 5.4
-     env: WP_VERSION=latest
+     env: WP_VERSION=latest WC_VERSION=2.6.14
    - php: 5.4
-     env: WP_VERSION=4.4
+     env: WP_VERSION=4.4 WC_VERSION=2.6.14
    - php: 5.5
-     env: WP_VERSION=latest
+     env: WP_VERSION=latest WC_VERSION=2.6.14
    - php: 5.5
-     env: WP_VERSION=4.4
+     env: WP_VERSION=4.4 WC_VERSION=2.6.14
+   - php: 5.5
+     env: WP_VERSION=latest WC_VERSION=3.0.3
      # 5.6 / master already included above as first build.
    - php: 5.6
-     env: WP_VERSION=4.4
+     env: WP_VERSION=4.4 WC_VERSION=2.6.14
+   - php: 5.6
+     env: WP_VERSION=4.4 WC_VERSION=3.0.3
```

Comment:
This was originally meant to test the latest and previous versions of WP - can you update `4.4` to `4.6.4`?

Also, why aren't we testing WC `3.0.3` against PHP versions < 5.5?